MahMaterialDragablzMashUp/PaletteSelectorViewModel.cs

```diff
@@ -1,38 +1,42 @@
-﻿using MaterialDesignColors;
+﻿using System;
+using MaterialDesignColors;
 using MaterialDesignThemes.Wpf;
 using System.Collections.Generic;
 using System.Windows.Input;
 
 namespace MahMaterialDragablzMashUp
 {
-    public class PaletteSelectorViewModel
+    public class PaletteSelectorViewModel : PaletteHelper
     {
         public PaletteSelectorViewModel()
         {
-            Swatches = new SwatchesProvider().Swatches;
-        }
-
-        public ICommand ToggleBaseCommand { get; } = new AnotherCommandImplementation(o => ApplyBase((bool)o));
-
-        private static void ApplyBase(bool isDark)
-        {
-            new PaletteHelper().SetLightDark(isDark);
+            Swatches = new SwatchesProvider().Swatches;  
+            ApplyPrimaryCommand = new DelegateCommand<Swatch>(ApplyPrimary, new Predicate<Swatch>(s => s != Primary));
```

Comment:
We can't automatically apply this in the demo, as it overrides App.xaml, and will confuse people who are creating a specific panel.  We'll need a button for people to explicitly click.